### ISSUE TITLE
Unable to start CRC 4.2.10 due to expired certs?

### ISSUE THREAD

**Issue by BenHall** [code-ready/crc#930]
Hello,

It looks like the certificates have expired for 4.2.10 and it is now unable to start. 

It gets to `INFO Cluster TLS certificates have expired, renewing them...` but seems to hang for a long time.

When will the next release be available?

cc @praveenkumar

**Comment by cfergeau** [comment]
Can you provide debug logs? Do you remember when was the last time the cluster successfully started?

**Comment by gbraad** [comment]
Renewing of the certs takes minutes... how long have you waited? (Has this been reproduced more than once?)

**Comment by praveenkumar** [comment]
@BenHall I tried today 1.3.0 release from scratch and `INFO Cluster TLS certificates have expired, renewing them...` step does take like 4-5 mins which also mentioned in the doc https://code-ready.github.io/crc/#troubleshooting-expired-certificates_gsg if you faced issue after that much time it still hanged or failed then please provide debug logs. 

> When will the next release be available?

We are trying to make it available it by coming Wednesday.

**Comment by BenHall** [comment]
Thanks. It was taking longer than expected, but when I switched to debug it was restarting the pods as expected.

Sorry for the trouble.

**Comment by gbraad** [comment]
I still believe we need to improve this experience by just indicating it is still ongoing; `progressdots`, @code-ready/crc-devel WDYT?

**Comment by cfergeau** [comment]
The log message is `logging.Infof("Cluster TLS certificates have expired, renewing them...")`. Are you suggesting animated progress dots?

**Comment by praveenkumar** [comment]
@cfergeau even not with animated progress dot can we just put `[ takes 4-5 mins ]` so the user know that he has to wait that long?

**Comment by BenHall** [comment]
@praveenkumar That would have helped and in line with starting of openshift when it takes 3 minutes.

**Comment by cfergeau** [comment]
The delay is not deterministic, what about `logging.Infof("This will take up to 5 minutes")` on a separate line?

**Comment by praveenkumar** [comment]
@cfergeau Sure that would also work or `logging.Infof("Cluster TLS certificates have expired, renewing them... [will take up to 5 mins]")` without adding a separate line?

**Comment by cfergeau** [comment]
Doesn't this make the line a bit too long?

**Comment by praveenkumar** [comment]
@cfergeau I don't have any strong opinion the way we put the line if you think adding a new line make it clear then sure :)

**Comment by chanseokoh** [comment]
Initially I thought this software is broken. After googling "readhat crc TLS expired renewing hangs" and reaching this issue to read all the comments, I decided to give another shot. It did work, but I think I waited more than 5 minutes.

**Comment by sterburg** [comment]
> Initially I thought this software is broken. After googling "readhat crc TLS expired renewing hangs" and reaching this issue to read all the comments, I decided to give another shot. It did work, but I think I waited more than 5 minutes.

Waiting more than 20 minutes now, still no progress .... :(
I'm in a disconnected and nested-virtualization (vmware) environment.

**Comment by cfergeau** [comment]
If you are still using 1.3.0/4.2.10, you will have to upgrade before we can try to help. If you are using something newer, please open a new issue, even though vmware+nested crc is often problematic.
